### ISSUE TITLE
toIso8601String()  was called on null.

### ISSUE THREAD

**Issue by arranf** [renefloor/flutter_cached_network_image#18]
```

I/flutter (20331): ══╡ EXCEPTION CAUGHT BY SERVICES ╞══════════════════════════════════════════════════════════════════
I/flutter (20331): The following NoSuchMethodError was thrown resolving an image codec:
I/flutter (20331): The method 'toIso8601String' was called on null.
I/flutter (20331): Receiver: null
I/flutter (20331): Tried calling: toIso8601String()
I/flutter (20331): 
I/flutter (20331): When the exception was thrown, this was the stack:
I/flutter (20331): #0      Object.noSuchMethod (dart:core-patch/dart:core/object_patch.dart:46)
I/flutter (20331): #1      CacheManager.getFile.<anonymous closure> (package:flutter_cache_manager/flutter_cache_manager.dart:224:76)
I/flutter (20331): <asynchronous suspension>
I/flutter (20331): #2      SynchronizedLock._run.<anonymous closure>.<anonymous closure> (package:synchronized/src/synchronized_impl.dart:87:29)
I/flutter (20331): #6      SynchronizedLock._run.<anonymous closure> (package:synchronized/src/synchronized_impl.dart:85:14)
I/flutter (20331): #8      SynchronizedLock._run (package:synchronized/src/synchronized_impl.dart:84:16)
I/flutter (20331): #9      SynchronizedLock.synchronized.run (package:synchronized/src/synchronized_impl.dart:149:14)
I/flutter (20331): #10     SynchronizedLock.synchronized (package:synchronized/src/synchronized_impl.dart:159:17)
I/flutter (20331): #11     synchronized (package:synchronized/src/synchronized_impl.dart:226:19)
I/flutter (20331): #12     synchronized (package:synchronized/synchronized.dart:32:10)
I/flutter (20331): #13     CacheManager.getFile (package:flutter_cache_manager/flutter_cache_manager.dart:194:11)
I/flutter (20331): <asynchronous suspension>
I/flutter (20331): #14     CachedNetworkImageProvider._loadAsync (package:cached_network_image/cached_network_image.dart:51:35)
I/flutter (20331): <asynchronous suspension>
I/flutter (20331): #15     CachedNetworkImageProvider.load (package:cached_network_image/cached_network_image.dart:40:16)
I/flutter (20331): #16     ImageProvider.resolve.<anonymous closure>.<anonymous closure> (package:flutter/src/painting/image_provider.dart:264:86)
I/flutter (20331): #17     ImageCache.putIfAbsent (package:flutter/src/painting/image_cache.dart:82:22)
I/flutter (20331): #18     ImageProvider.resolve.<anonymous closure> (package:flutter/src/painting/image_provider.dart:264:63)
I/flutter (20331): #19     SynchronousFuture.then (package:flutter/src/foundation/synchronous_future.dart:38:29)
I/flutter (20331): #20     ImageProvider.resolve (package:flutter/src/painting/image_provider.dart:262:30)
I/flutter (20331): #21     _ImageState._resolveImage (package:flutter/src/widgets/image.dart:484:20)
I/flutter (20331): #22     _ImageState.didChangeDependencies (package:flutter/src/widgets/image.dart:459:5)
I/flutter (20331): #23     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3660:12)
I/flutter (20331): #24     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #25     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #26     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #27     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #28     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #29     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #30     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #31     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #32     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #33     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #34     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #35     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #36     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3662:22)
I/flutter (20331): #37     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #39     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #40     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #41     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #42     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #43     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #44     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #45     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #46     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #47     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #48     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #49     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #50     ParentDataElement.mount (package:flutter/src/widgets/framework.dart:3837:16)
I/flutter (20331): #51     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #52     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4637:32)
I/flutter (20331): #53     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #54     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4637:32)
I/flutter (20331): #55     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #56     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #57     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #58     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #59     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #60     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #61     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #62     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #63     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #64     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #65     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #66     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #67     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #68     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #69     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #70     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #71     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #72     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #73     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #74     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #75     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3662:22)
I/flutter (20331): #76     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #77     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #78     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #79     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #80     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #81     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #82     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #83     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #84     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #85     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #86     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #87     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #88     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #89     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #90     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #91     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #92     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #93     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #94     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3662:22)
I/flutter (20331): #95     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #96     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #97     Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #98     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #99     Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #100    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #101    StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3662:22)
I/flutter (20331): #102    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #103    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #104    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #105    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #106    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #107    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #108    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #109    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #110    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #111    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #112    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #113    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #114    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #115    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #116    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #117    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #118    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #119    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #120    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #121    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #122    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #123    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #124    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #125    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #126    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #127    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #128    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #129    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #130    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #131    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #132    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #133    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #134    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #135    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #136    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #137    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #138    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4532:14)
I/flutter (20331): #139    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #140    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #141    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #142    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #143    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #144    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #145    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #146    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #147    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #148    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #149    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #150    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #151    ParentDataElement.mount (package:flutter/src/widgets/framework.dart:3837:16)
I/flutter (20331): #152    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #153    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #154    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3556:16)
I/flutter (20331): #155    Element.rebuild (package:flutter/src/widgets/framework.dart:3445:5)
I/flutter (20331): #156    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3524:5)
I/flutter (20331): #157    StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3662:22)
I/flutter (20331): #158    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3519:5)
I/flutter (20331): #159    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2857:14)
I/flutter (20331): #160    Element.updateChild (package:flutter/src/widgets/framework.dart:2660:12)
I/flutter (20331): #161    SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:723:20)
I/flutter (20331): #162    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2173:19)
I/flutter (20331): #163    SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:716:11)
I/flutter (20331): #164    RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:238:23)
I/flutter (20331): #165    RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:1647:58)
I/flutter (20331): #166    PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:734:15)
I/flutter (20331): #167    RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:1647:13)
I/flutter (20331): #168    RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:227:5)
I/flutter (20331): #169    RenderSliverMultiBoxAdaptor.addInitialChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:335:5)
I/flutter (20331): #170    RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:78:12)
I/flutter (20331): #171    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #172    RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:182:11)
I/flutter (20331): #173    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #174    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:287:13)
I/flutter (20331): #175    RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:980:12)
I/flutter (20331): #176    RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:904:20)
I/flutter (20331): #177    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #178    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #179    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #180    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #181    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #182    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #183    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #184    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #185    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #186    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #187    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #188    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #189    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #190    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #191    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #192    RenderSliverFixedExtentBoxAdaptor.performLayout (package:flutter/src/rendering/sliver_fixed_extent_list.dart:165:18)
I/flutter (20331): #193    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #194    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:287:13)
I/flutter (20331): #195    RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:980:12)
I/flutter (20331): #196    RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:904:20)
I/flutter (20331): #197    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #198    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #199    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #200    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #201    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #202    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #203    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #204    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #205    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #206    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #207    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #208    RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
I/flutter (20331): #209    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #210    RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:798:17)
I/flutter (20331): #211    RenderObject.layout (package:flutter/src/rendering/object.dart:1551:7)
I/flutter (20331): #212    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:124:11)
I/flutter (20331): #213    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:92:7)
I/flutter (20331): #214    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:194:7)
I/flutter (20331): #215    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:338:14)
I/flutter (20331): #216    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1426:7)
I/flutter (20331): #217    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:712:18)
I/flutter (20331): #218    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:265:19)
I/flutter (20331): #219    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:612:22)
I/flutter (20331): #220    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:203:5)
I/flutter (20331): #221    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:916:15)
I/flutter (20331): #222    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:856:9)
I/flutter (20331): #223    BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:768:5)
I/flutter (20331): #224    _invoke (file:///b/build/slave/Linux_Engine/build/src/flutter/lib/ui/hooks.dart:113)
I/flutter (20331): #225    _drawFrame (file:///b/build/slave/Linux_Engine/build/src/flutter/lib/ui/hooks.dart:102)
I/flutter (20331): (elided 4 frames from package dart:async)
I/flutter (20331): 
I/flutter (20331): Image provider:
I/flutter (20331): CachedNetworkImageProvider("https://s3.eu-west-1.amazonaws.com/data.heroescompanion.com/images/talents/storm_ui_icon_maiev_memento.png",
I/flutter (20331): scale: 1.0)
I/flutter (20331): Image key:
I/flutter (20331): CachedNetworkImageProvider("https://s3.eu-west-1.amazonaws.com/data.heroescompanion.com/images/talents/storm_ui_icon_maiev_memento.png",
I/flutter (20331): scale: 1.0)
I/flutter (20331): ════════════════════════════════════════════════════════════════════════════════════════════════════
```

**Comment by renefloor** [comment]
Hmm.. This should have been fixed in flutter_cache_manager 0.0.4+1, which should automatically be used by this package. Can you do `flutter packages upgrade` and try to see if that fixes it?

**Comment by arranf** [comment]
I *believe* this issue has been fixed by running `flutter packages upgrade` but I am still getting the occasional exception via Sentry but I suspect they are from older versions.
